### PR TITLE
Abbrevation

### PR DESCRIPTION
**Pull request by andrewkatz**
@trobrock

### CODE REVIEW

**Comment by andrewkatz** [comment]
@trobrock Can you take a look at this? Just added the `zero?` method.

File: lib/weighable/weight.rb [hunk 1]
```diff
@@ -185,6 +185,10 @@ def round(precision = 0)
       self
     end
 
+    def zero?
+      @value == 0
```
Comment (trobrock) [inline lib/weighable/weight.rb]: might just want to be `@value.zero?`

**Comment by trobrock** [comment]
👍

File: lib/weighable/weight.rb (hunk 1 above)
Comment (andrewkatz) [reply lib/weighable/weight.rb]: Oh cool, didn't know BigDecimal had that. I'll update this.

**Pull request merged**